src/lando/utils/management/commands/generate_version_file.py

```diff
@@ -0,0 +1,8 @@
+from django.core.management.base import BaseCommand
+from setuptools_scm import get_version
+
+class Command(BaseCommand):
+    help = "Explicitly generate the 'version.py' file."
+
+    def handle(self, *args, **options):
+        get_version(write_to="src/lando/version.py")

```

Comment:
nit: I think it would be slightly better to do `get_version(write_to=settings.BASE_DIR / "version.py")`. That way fewer things are hardcoded.

Reply:
I don't think this works since it's expecting a relative path. The error is:

```
        for step, path in enumerate([other] + list(other.parents)):
            if self.is_relative_to(path):
                break
            elif not walk_up:
>               raise ValueError(f"{str(self)!r} is not in the subpath of {str(other)!r}")
E               ValueError: '/code/src/lando/version.py' is not in the subpath of '.'
```
Since `.` is `code` already, it can't resolve `/code/`.

```
get_version(write_to=Path(*settings.BASE_DIR.parts[2:]) / "version.py")
```
Would work, but it's less clear than what it is now, so I think leaving it as-is is best.